extern_test.rs

```diff
@@ -0,0 +1,13 @@
+extern crate creusot_contracts;
+use creusot_contracts::*;
+
+#[creusot::no_translate]
+#[creusot::extern_spec]
```

Comment:
Would it be possible to make `extern_spec` imply `no_translate` automatically?

Reply:
Yes, one of the final steps will be to wrap this all up in a macro. What you're seeing here is the desugared syntax